genum/gen/generate.go

```diff
@@ -263,3 +307,18 @@ func isDeprecated(fAST *ast.File, name string) bool {
 	}
 	return false
 }
+
+func (g *Generate) ConvertibleFromString(inputType types.Type) bool {
+	return types.ConvertibleTo(stringType, inputType)
+}
+
+func (g *Generate) ConvertibleFromInt(inputType types.Type) bool {
```

Comment:
These look like they should be methods of a trait, especially based on the template usage.

Reply:
yup i have moved them